test/blackbox-tests/test-cases/mdx-stanza/run.t

```diff
@@ -9,11 +9,12 @@ It also requires dune lang 2.4 or higher
 
   $ dune build @install --root lang-version/
   Entering directory 'lang-version'
-  File "dune", line 1, characters 0-5:
-  1 | (mdx)
-      ^^^^^
-  Error: 'mdx' is only available since version 2.4 of the dune language. Please
-  update your dune-project file to have (lang 2.4).
+  File "dune-project", line 3, characters 11-14:
+  3 | (using mdx 0.1)
+                 ^^^
+  Error: Version 0.1 of mdx is not supported until dune lang 2.4.
```

Comment:
I slightly preferred the old wording here, i.e. `version 2.4 of the dune language` rather than `dune lang 2.4`.